src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -336,6 +341,15 @@ public void periodic() {
         "Elevator/Motor1/Velocity", elevatorMotor1.getVelocity().getValue().in(DegreesPerSecond));
     Logger.recordOutput(
         "Elevator/Motor1/Position (Degrees)", elevatorMotor1.getPosition().getValue().in(Degrees));
+    Logger.recordOutput(
+        "Elevator/Motor1/Stator Current (Amps)",
+        elevatorMotor1.getStatorCurrent().getValueAsDouble());
+    Logger.recordOutput(
+        "Elevator/Motor1/Supply Current (Amps)",
+        elevatorMotor1.getSupplyCurrent().getValueAsDouble());
+    Logger.recordOutput(
+        "Elevator/Motor1/Stall Current (Amps)",
+        elevatorMotor1.getMotorStallCurrent().getValueAsDouble());
```

Comment:
I think we are mainly interested in the stator current, the other ones can go away